src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+
+import usersFromServer from '../../api/users';
+import { getUserById } from '../../services/user';
+import { Todo } from '../../Types/Todo';
+
+type Props = {
+  onSubmit: (post: Todo) => void;
+};
+
+export const NewTodo: React.FC<Props> = ({ onSubmit }) => {
+  const [title, setTitle] = useState('');
+  const [hasTitleErorr, setHasTitleErorr] = useState(false);
+
+  const [userId, setUserId] = useState(0);
+  const [hasIserIdErorr, setHasIserIdErorr] = useState(false);
```

Comment:
Typo found in variable name 'hasIserIdErorr'. Consider renaming to 'hasUserIdError'.